Proofs/Experiments/Memcpy/MemCpyVCG.lean

```diff
@@ -741,19 +741,19 @@ theorem partial_correctness :
           apply zero_iff_z_eq_one
         simp only [h_s5_z]
 
-        simp only [show s5.x0 ≤ s0.x0 by bv_omega, true_and]
+        simp only [show s5.x0 ≤ s0.x0 by bv_omega_bench, true_and]
         rw [h_s5_x0, h_s5_x1, h_si_x1]
         have h_s0_x1 : s0.x1 + 0x10#64 * (s0.x0 - si.x0) + 0x10#64 = s0.x1 + 0x10#64 * (s0.x0 - (si.x0 - 0x1#64)) := by
-          rw [show s0.x0 - (si.x0 - 0x1#64) = (s0.x0 - si.x0) + 0x1#64 by skip_proof bv_omega,
+          rw [show s0.x0 - (si.x0 - 0x1#64) = (s0.x0 - si.x0) + 0x1#64 by skip_proof bv_omega_bench,
```

Comment:
What does `skip_proof bv_omega(_bench)` do?

Reply:
`skip_proof` is a combinator I wrote that replaces the proof with what is morally a `sorry`, if the option `skip_proof`is enabled (it is disabled by default, I wrote it to make writing proofs tolerable).

`bv_omega_bench` runs `bv_omega`, and also outputs benchmarking info into a text file.